pages/en/partials/class-electronic.md

```diff
@@ -6,3 +6,100 @@ readMore:   class-electronic
+- It is recommended that the course be undertaken by children 12 years and older
+- It is assumed kids will have previously attended Scratch/Web CoderDojo sessions
+- Have a willingness to learn and have fun :-)
+
+## When
+- The course is run during school term every Saturday from 11am-1pm.
+- Any Saturdays when the class isn't running will be announced in advance.
+
+## Labs
+Various hands on labs will be conducted throughout the year including:
+- Use "Hot wires" snap circuits to build circuits
+- Take apart a computer - understand various internal parts
+- Brining in old toys with motors
+- Create circuits
+- Tinkercad to create online circuits
+- Learn about Microcontrollers using arduino labs (Dojo sushis)
+    * Blinking LED
+    * Traffic Lights
+    * Temperature Sensors
+    * Ultrasonic Range Sensor
+    * LCDs
+    * Motors
+- Rasbperry Pi introduction
+  - Basic introduction to Rasbian
+  - Physical computing with Scratch (https://projects.raspberrypi.org/en/projects/physical-computing-with-scratch
+  - Finding your way around a Raspberry Pi and using a shell
+  - Basic linux commands
+  - Using Pi to perform physical computing
+
+## Group Projects
+
+- Kids can enter [Coolest Project](https://coolestprojects.org/)
+- Kids break up into groups come up with ideas for projects themselves. Alternatively perhaps mentors can
+assist coming up with a project.
+- Some assistance from parents or the club may be required if the team want to use equipment we don't currently have in stock
+- If the kids want to build something. Mentors will assist.
+
+## References / Resources
+
+* [Arduino resources](http://kata.coderdojo.com/wiki/Arduino_Path) Dojo Kata
+* [Raspberry Pi resources](https://curriculum.raspberrypi.org/physical-computing/creator/)
+* Coolect Projects [FAQ](https://help.coolestprojects.org/hc)
+* [Coder Dojo resources](https://coderdojo.com/resources/)
+
+## House rules
+
+- Kids under 13 must be accompanied by an adult
+- Kids need to register their intention to join the group each week
+- No food or drink permitted in the electronics lab
+- Please be careful with the equipment you use - it the property of AIT or a volunteer
+- Leave the venue in the same condition you found it in
+- Tidy up and shutdown workstations when the session is over
+- Be cool to your peers!
+- Get to know the people in your group and HAVE FUN !
+
+## What to bring
+
+- Kids are welcome to bring along their own electronics kits, if they have them.
+  - Breadboard, Resistors, LEDs
+  - Arduino , Raspberry Pi
+- Mentors will offer recommendations on kits to purchase if parents wish to do this.
+- The group mentors have a limited supply of electronic equipment which the group can use / share
+- For those kids who bring their own kits, they take responsibility for looking after their equipment.
+
+## Other things to note
+
+- If a quota of volunteers are not available in any particular week the course may not run or will run in a limited capacity
```

Comment:
Same here, good thoughts applicable to all classes

Reply:
I suggest to leave this statement here as it is specific to the electronics group, as there may weeks where no mentors are available so the electronics dojo may not run at all.

Reply:
ok